### DISCUSSION TITLE
Can appium for Windows app run properly on Windows Server 2016?

### DISCUSSION THREAD

**Discussion by ricardoleonhart** [appium/appium#16744] [Q&A]
In this link https://appium.io/docs/en/drivers/windows/, it mentions appium is using WinAppDriver and that the requirements for it are
- Windows PC with Windows 10 or up
- Ability to enter Administrator mode

However, when I check the requirements for WinAppDriver, it mentions that WS2016 (Windows Server 2016) is supported: https://github.com/microsoft/WinAppDriver/blob/master/Docs/FAQ.md

My aim is to deploy and run Windows app UI tests on a WS2016 machine and so far, it seems to be working based on some simple tests that I created. 

However, the fact that WS2016 isn't mentioned in the first link above makes me feel a little uneasy because although my simple tests worked fine, there could be issues when they get more complicated by which it would be really expensive to use another UI testing framework.

Am I just being paranoid? Or does anyone know if Appium actually do support WS2016? Anecdotal experiences are welcomed.

**Comment by KazuCocoa** [comment]
Perhaps no one in Appium team haven't tested the environment.
Theoretically, if WinAppDriver works on the environment (and nodejs environment works), appium/windows driver should work.
